sonic-thermalctld/scripts/thermalctld

```diff
@@ -515,9 +524,13 @@ class TemperatureUpdater(logger.Logger):
         low_threshold = NOT_AVAILABLE
         high_critical_threshold = NOT_AVAILABLE
         low_critical_threshold = NOT_AVAILABLE
+        maximum_temperature = NOT_AVAILABLE
+        minimum_temperature = NOT_AVAILABLE
         temperature = try_get(thermal.get_temperature)
         if temperature != NOT_AVAILABLE:
             temperature_status.set_temperature(name, temperature)
+            minimum_temperature = try_get(thermal.get_minimum_recorded)
+            maximum_temperature = try_get(thermal.get_maximum_recorded)
```

Comment:
What precisely 'recorded' here implies?
Temperature value fetched on a sensor is a particular value at a given time, so trying to see what's max and min here?

Reply:
As we sync'ed up in 10/21 Chassis Workgroup meeting, please make this optional as not all vendors/platforms support this

Reply:
The try_get() wrapper makes it optional.